bz#752559 - Disable the dialog on start

Reported ans accepted upstream as https://bugs.wireshark.org/bugzilla/show_bug.cgi?id=6559

diff --git a/gtk/simple_dialog.c b/gtk/simple_dialog.c
index ff7d283..8ee40cf 100644
--- a/gtk/simple_dialog.c
+++ b/gtk/simple_dialog.c
@@ -316,10 +316,11 @@ vsimple_dialog(ESD_TYPE_E type, gint btn_mask, const gchar *msg_format, va_list
     state = gdk_window_get_state(top_level->window);
   }
 
-  /* If we don't yet have a main window or it's iconified, don't show the
-     dialog. If showing up a dialog, while main window is iconified, program
-     will become unresponsive! */
-  if (top_level == NULL || state & GDK_WINDOW_STATE_ICONIFIED) {
+  /* If we don't yet have a main window or it's iconified or hidden (i.e. not
+     yet ready, don't show the dialog. If showing up a dialog, while main
+     window is iconified, program will become unresponsive! */
+  if (top_level == NULL || state & GDK_WINDOW_STATE_ICONIFIED
+          || state & GDK_WINDOW_STATE_WITHDRAWN) {
 
     queued_message = g_malloc(sizeof (queued_message_t));
     queued_message->type = type;
